crates/vsock/src/txbuf.rs

```diff
@@ -18,14 +18,19 @@ pub(crate) struct LocalTxBuf {
 
 impl LocalTxBuf {
     /// Create a new instance of LocalTxBuf.
-    pub fn new() -> Self {
+    pub fn new(local_tx_buf_size: u32) -> Self {
```

Comment:
what about just `size` or `buf_size`? The struct is already called `LocalTxBuf`.